src/actions/getDatasetItems.js

```diff
@@ -45,13 +42,13 @@ export default async function getDatasetItems(apifyClient, maybeDatasetId) {
     if (dataset.itemCount > RESULTS_FILE_LIMIT) {
```

Comment:
Do we need this `if`? Can we just put manifest creation to `if (...) {`?

Reply:
It was recommended by keboola guys, when we have a small number of results we have to create the only file with headers row otherwise we have to create a folder with csv chunks without header row and header row put in manifest instead